WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -53,6 +53,18 @@ final class OrderDetailsDataSource: NSObject {
             !orderContainsAnySubscription()
     }
 
+    var isEligibleForRefund: Bool {
+        guard refundableOrderItemsDeterminer.isAnythingToRefund(from: order, with: refunds, currencyFormatter: currencyFormatter),
+              let orderTotal = Double(order.total),
+              orderTotal > 0,
```

Comment:
nit: maybe these two lines can be moved to the determiner, like as the first check? 😄

Reply:
Yeah, I did that, I moved them to the `OrderRefundsOptionsDeterminer`